busybox/busybox-pvc.yaml

```diff
@@ -6,8 +6,9 @@ metadata:
   labels:
     appname: busybox
 spec:
-  accessModes: [ "ReadWriteOnce" ]
-  storageClassName: "rook-ceph-block"
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: rook-ceph-block
   resources:
     requests:
-      storage: 1Gi
+      storage: 5Gi
```

Comment:
Why the change in size?

Reply:
All the other variants use 5Gi - we can change this in the kustomization but I don't see a reason to invest time in keeping different size. If there is no good reason to use 5Gi we can change all of them to use 1Gi.

Reply:
I changed all pvcs to 1Gi since it makes more sense. If there is real need to use different size we can modify it later, maybe only on the specific variant that need this.